src/main/java/org/worldcubeassociation/ui/ExportJSONAction.java

```diff
@@ -45,7 +46,7 @@ public void actionPerformed(ActionEvent aActionEvent) {
         }
 
         try {
-            String jsonFileName = "Results for " + getEnv().getCompetitionId() + ".json";
+            String jsonFileName = "Results (" + WorkbookAssistant.class.getPackage().getImplementationVersion() + ") for " + getEnv().getCompetitionId() + ".json";
```

Comment:
Oh, well, in this case, a field inside the JSON and its content shown when exporting would be awesome.